src/app/components/question-approval-page/question-approval-page.component.html

```diff
@@ -0,0 +1,88 @@
+<clr-main-container class="main-container">
+    <clr-header class="header header-1">
+        <div class="branding">
+            <a routerLink="/home"  routerLinkActive="active" class="nav-link">
+                <img src="assets/dojo-transparent-white-outline.svg" alt="A dojo" class="mtdj__logo">
+                <span class="title">Math Dojo</span>
+            </a>
+        </div>
+    </clr-header>
+    <nav class="subnav">
+        <ul class="nav">
+            <li class="nav-item">
+                <a class="nav-link" routerLink="/actions">Actions</a>
+            </li>
+            <li class="nav-item">
+               > Question Queue
```

Comment:
Rather than refer to the Queue, which is our technology choice, better to go for the intent, which is Quarantine. Or in this case because they are actions related to Questions as a whole, perhaps Actions > Questions > Quarantine .